proto-buf/services/eigentrust.proto

```diff
@@ -0,0 +1,285 @@
+syntax = "proto3";
+package eigentrust;
+
+import "google/protobuf/any.proto";
+
+option go_package = "k3l.io/go-eigentrust/api/pb";
+
+// A timestamp is expressed in one or more qwords (64-bit unsigned integers),
+// and is used for trust matrices/vectors and their entries.
+// go-eigentrust treats the qwords as a variable-length big endian unsigned int,
+// e.g. {0x1234, 0x567890abcdef} -> 0x12340000567890abcdef.
+//
+// Their semantics are to be defined by applications.  Examples:
+//
+// - {seconds} of a UNIX timestamp
+// - {microseconds} of a UNIX timestamp
+// - {seconds, nanosecond part (0-999999999)} of a UNIX timestamp
+// - {block height, txn index} of an EVM on-chain event.
+
+message TrustMatrixHeader {
+  optional string id = 1;
+  repeated uint64 timestamp_qwords = 2;
+}
+
+message TrustMatrixEntry {
+  string truster = 1;
+  string trustee = 2;
+  double value = 3;
+}
```

Comment:
This should also include timestamp, no?

Reply:
No, the timestamp is in the header.  The stream contains one header part, followed by zero or more entry parts.  All the entries share the same timestamp in the header preceding them.

Reply:
Roger, fixing.